orttraining/orttraining/python/training/orttrainer_options.py

```diff
@@ -395,7 +395,7 @@ class ORTTrainerOptions(object):
             enable PyTorch to export nodes as contrib ops in ONNX.
             This flag may be removed anytime in the future.
         session_options (onnxruntime.SessionOptions):
-            The SessionOptions instance that TrainingSession will use.
+            The SessionOptions instance that TrainingSession will use .
```

Comment:
typo?

Reply:
no, just a minor update to trigger the re-sync on the CI... otherwise it pick some old configuration...